packages/genesys-spark-components/src/components/stable/gux-icon/example.html

```diff
@@ -7,6 +7,7 @@ <h1>gux-icon</h1>
     <gux-tab tab-id="1-3">Legacy icons</gux-tab>
     <gux-tab tab-id="1-4">Restricted icons</gux-tab>
     <gux-tab tab-id="1-5">Deprecated icons</gux-tab>
+    <gux-tab tab-id="1-6">COMUI-2696 icons</gux-tab>
```

Comment:
The changes in this file were only added to simplify the PR review process. Once we are happy with the change I will remove them before merging